### PR TITLE
Add template

### PR DESCRIPTION
**Pull request by kwongjose**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# XscapeVelocity-Big-Data
\ No newline at end of file
+# XscapeVelocity-Big-Data
+John Caspers
```
Comment (clarkemily) [inline README.md]: MERGE CONFLICT

**Review by clarkemily — APPROVED** [review]

File: Docs/template.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	"poc":  1,
+	"latitude": 33.2342,
+	"longitude": 32.2342,
+	"datum": "string",
+	"parameterName": "string",
+	"dataLocal": "string",
+	"timeLocal": "string",
+	"dateGMT": "string",
+	"timeGMT": "string",
+	"sampleMeasurment": 3.1,
```
Comment (clarkemily) [inline Docs/template.json]: sampleMeasurement

File: Docs/template.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+	"UnitOfMeasure": "string",
```
Comment (clarkemily) [inline Docs/template.json]: unitOfMeasure

File: Docs/template.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
+	"stateCode": "string",
+	"countyCode": "string",
+	"siteNum": "string",
+	"poc":  1,
+	"latitude": 33.2342,
+	"longitude": 32.2342,
+	"datum": "string",
+	"parameterName": "string",
+	"dataLocal": "string",
```
Comment (clarkemily) [inline Docs/template.json]: dateLocal

**Review by Andrew-Bros — APPROVED** [review]

**Review by clarkemily — APPROVED** [review]

**Review by jackcaspers — APPROVED** [review]

**Pull request merged**